### PR TITLE
magento/devdocs#: Adding item to the list

### PR DESCRIPTION
**Pull request by DashukVolodymyr**
## Purpose of this pull request

This pull request (PR) adds an item to the list

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.4/install-gde/install/sample-data-after-clone.html#sample-next-steps
- https://devdocs.magento.com/guides/v2.3/install-gde/install/sample-data-after-clone.html#sample-next-steps

## Links to Magento source code

If you are fixing a GitHub issue, note it using GitHub keyword format (https://help.github.com/en/articles/closing-issues-using-keywords#closing-an-issue-in-a-different-repository) to close the issue when this pull request is merged. Example: `Fixes #1234`

`master` is the default branch. Merged pull requests to `master` go live on the site automatically. Any requested changes to content on the `master` branch must be related to the released codebase. Any content related to future releases goes in the `develop` branch.

See Contribution guidelines (https://github.com/magento/devdocs/blob/master/.github/CONTRIBUTING.md) for more information.
-->

### CODE REVIEW

File: src/_includes/install/sampledata/sample-data-clone-24.md [hunk 1]
```diff
@@ -87,6 +87,10 @@ To clone the Magento sample data GitHub repository using the SSH protocol:
 
 1. See [Set file system permissions and ownership](#samp-data-perms).
 
+1. ```bash
```
Comment (atwixfirster) [inline src/_includes/install/sampledata/sample-data-clone-24.md:90]: In the `point 8` we've changed a directory to `<magento_root>`.

Please add the next for `point 12`:

Run the following command:

```bash
bin/magento setup:upgrade
```

Thank you!

File: src/_includes/install/sampledata/sample-data-clone.md [hunk 2]
```diff
@@ -87,6 +87,10 @@ To clone the Magento sample data GitHub repository using the SSH protocol:
 
 1. See [Set file system permissions and ownership](#samp-data-perms).
 
+1. ```bash
```
Comment (atwixfirster) [inline src/_includes/install/sampledata/sample-data-clone.md:90]: The same here.

Thank you!

File: src/_includes/install/sampledata/sample-data-clone-24.md (hunk 1 above)
Comment (DashukVolodymyr) [reply src/_includes/install/sampledata/sample-data-clone-24.md]: Hi @atwixfirster 
Thanks for your notes.

Everything is done at your request.
Could you please proceed with the code review?

File: src/_includes/install/sampledata/sample-data-clone.md (hunk 2 above)
Comment (DashukVolodymyr) [reply src/_includes/install/sampledata/sample-data-clone.md]: @atwixfirster 
Thanks for your notes.

Everything is done at your request.
Could you please proceed with the code review?

File: src/_includes/install/sampledata/sample-data-clone.md [hunk 3]
```diff
@@ -87,6 +87,11 @@ To clone the Magento sample data GitHub repository using the SSH protocol:
 
 1. See [Set file system permissions and ownership](#samp-data-perms).
 
+1. Run the following command:
+   ```bash
```
Comment (atwixfirster) [inline src/_includes/install/sampledata/sample-data-clone.md:91]: Add an empty line before ```bash

Thank you!

File: src/_includes/install/sampledata/sample-data-clone-24.md (hunk 1 above)
Comment (atwixfirster) [reply src/_includes/install/sampledata/sample-data-clone-24.md]: Almost done.

Comment (DashukVolodymyr) [reply src/_includes/install/sampledata/sample-data-clone-24.md]: @atwixfirster I added an empty line.
Please proceed.
Thank you!

File: src/_includes/install/sampledata/sample-data-clone-24.md [hunk 4]
```diff
@@ -87,6 +87,11 @@ To clone the Magento sample data GitHub repository using the SSH protocol:
 
 1. See [Set file system permissions and ownership](#samp-data-perms).
 
+1. Run the following command:
+   ```bash
```
[the remark this answers is not in the document]
Comment (DashukVolodymyr) [reply src/_includes/install/sampledata/sample-data-clone-24.md:92]: @atwixfirster I added an empty line.
Please proceed.
Thank you!

File: src/_includes/install/sampledata/sample-data-clone.md (hunk 3 above)
Comment (DashukVolodymyr) [reply src/_includes/install/sampledata/sample-data-clone.md:92]: @atwixfirster I added an empty line.
Please proceed.
Thank you!

Comment (atwixfirster) [reply src/_includes/install/sampledata/sample-data-clone.md:92]: Looks ok to me. Thank you, @DashukVolodymyr 👍 

@rogyar , could you please review this PR as a maintaner.

Thank you!

**Review by rogyar — APPROVED** [review]
Hi @DashukVolodymyr. Thank you for picking up my suggestion. Good job!

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
running tests

**Pull request merged**